### PR TITLE
enable sprout for jenkins run. 

### PR DESCRIPTION
**Pull request by amavinag**
--use-sprout can be used to provision appliance
        from sprout. the lease time for performance test
        appliance would be 1 week by default.
  --appliance-name also can be specified for py.test
        commandline option along with --appliance

### CODE REVIEW

**Comment by amavinag** [comment]
log output look like this for test run:
2016-07-07 10:14:21,300 : MainThread  :    INFO : Appliance IP is 1.1.1.1 (conftest.py:30)
2016-07-07 10:14:21,301 : MainThread  :    INFO : Appliance name is test_name (conftest.py:31)

File: cfme-performance/conftest.py [hunk 1]
```diff
@@ -15,16 +20,78 @@ def logger():
...
+                    pool_id, str(config.option.sprout_desc))
+            try:
+                result = wait_for(
+                    lambda: sprout_client.request_check(sprout_pool)["fulfilled"],
+                    num_sec=3600,
+                    delay=5,
+                    message="requesting appliances was fulfilled"
+                )
+            except:
+                pool = sprout_client.request_check(sprout_pool)
```
Comment (akrzos) [inline cfme-performance/conftest.py]: Looks like this is assigned but not used.

File: cfme-performance/utils/sprout.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import json
+import os
```
Comment (akrzos) [inline cfme-performance/utils/sprout.py]: imported but unused

File: cfme-performance/utils/sprout.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import requests
+
+from utils.conf import cfme_performance
+from utils.wait import wait_for
```
Comment (akrzos) [inline cfme-performance/utils/sprout.py]: Do we need utils/wait.py for this to import?

**Pull request merged**